org.tuxfamily.StuntRally.json

```diff
@@ -0,0 +1,192 @@
+                    "type": "archive",
+                    "url": "https://zlib.net/zlib-1.2.11.tar.gz",
+                    "sha256": "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1"
+                }
+            ]
+        },
+        {
+            "name": "zziplib",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/gdraheim/zziplib/archive/v0.13.69.tar.gz",
+                    "sha256": "846246d7cdeee405d8d21e2922c6e97f55f24ecbe3b6dcf5778073a88f120544"
+                }
+            ]
+        },
+        {
+            "name": "OGRE",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DOGRE_BUILD_COMPONENT_JAVA=FALSE"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/OGRECave/ogre/archive/v1.11.0.zip",
+                    "sha256": "480d2c47c317d1797a1b6f104a65283f6494ac680740a909163982f4ab0cef82"
+                }
+            ]
+        },
+        {
+            "name": "Boost",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.gz",
+                    "sha256": "8aa4e330c870ef50a896634c931adf468b21f8a69b77007e45c444151229f665"
+                }
+            ],
+            "build-commands": [
+                "./bootstrap.sh",
+                "./b2 install variant=release link=shared runtime-link=shared --prefix=/app --with-system --with-filesystem --with-thread --with-wave -j `nproc`"
+            ]
+        },
+        {
+            "name": "Bullet",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DBUILD_PYBULLET=OFF",
+                "-DBUILD_PYBULLET_NUMPY=OFF",
+                "-DUSE_DOUBLE_PRECISION=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/bulletphysics/bullet3/archive/2.87.tar.gz",
+                    "sha256": "438c151c48840fe3f902ec260d9496f8beb26dba4b17769a4a53212903935f95"
+                }
+            ]
+        },
+        {
+            "name": "OIS",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/wgois/OIS/archive/v1.4.tar.gz",
+                    "sha256": "aaa9289a0cc0bffbb03e894b1d10e65eac14572a2282d539afe48aec1ae35c75"
+                }
+            ]
+        },
+        {
+            "name": "MyGUI",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/MyGUI/mygui.git",
+                    "commit": "3687a78139837b598fbe703f2833add4c39adcca"
```

Comment:
I had to use a commit from master here. The last stable version of MyGUI is from 2015, is not compatible with newer Ogre3D and fails to compile. The stable version would have to be heavily patched to make it working correctly. :-/